quantum/config_common.h

```diff
@@ -303,6 +303,25 @@
                 UCSR1C = _BV(UCSZ11) | _BV(UCSZ10); \
                 sei();                              \
             } while (0)
+#   elif (defined(__AVR_AT90USB646__) || defined(__AVR_AT90USB1286__))
+#      define SERIAL_UART_BAUD 115200
+#      define SERIAL_UART_DATA UDR1
+       /* UBRR should result in ~16 and set UCSR1A = _BV(U2X1) as per rn42 documentation. HC05 needs baudrate configured accordingly */
+#      define SERIAL_UART_UBRR (F_CPU / (8UL * SERIAL_UART_BAUD) - 1)
```

Comment:
Sorry I only have the at90usb1286 and a hc-05 flashed with rn42 firmware.

http://ww1.microchip.com/downloads/en/DeviceDoc/bluetooth_cr_UG-v1.0r.pdf
The default values of rn42 can be found in Appendix A of the document and is 115K so this is what I have been using.

I thing in general baud rate should be moved to keyboard settings since people might have different adapters or settings anyway.

Reply:
Yeah, seems like this code needs a bit of love. Grepping through the repo for `SERIAL_UART_BAUD` reveals a handful of redefinitions of this whole config block for various keyboards, with various baudrates. So if it's working well for you, then this is good to go, I think. We can make it more general later.